packages/common/src/codegen/modules.d.ts

```diff
@@ -1 +1 @@
-declare module "prettier-plugin-solidity";
+declare module "prettier-plugin-solidity" {}
```

Comment:
The tsup's internal dts generation process doesn't support the original syntax, so I've changed it to a format that both tsc and tsup accept. There is no change in meaning in this context. Without this, the following error occurs on tsup:

```
Error: namespace must have a "ModuleBlock" body.

> 1 | declare module "prettier-plugin-solidity";
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```